producer/nf_management.go

```diff
@@ -329,53 +346,59 @@ func sendNFDownNotification(nfProfile models.NfProfile, nfInstanceID string) {
 	}
 }
 
-func UpdateNFInstanceProcedure(nfInstanceID string, patchJSON []byte) (response map[string]interface{}) {
+func UpdateNFInstanceProcedure(nfInstanceID string, patchJSON []byte) (response map[string]interface{}, err error) {
+	// Validation for NF Instance ID
+	if nfInstanceID == "" {
+		logger.ManagementLog.Errorln("NF Instance ID is required")
```

Comment:
Lowere case `nf` is used in loggings below
```suggestion
		logger.ManagementLog.Errorln("nf Instance ID is required")
```